CodeEdit/WorkspaceView.swift

```diff
@@ -28,7 +28,8 @@ struct WorkspaceView: View {
     var body: some View {
         NavigationView {
             if workspace.workspaceClient != nil {
-                NavigatorSidebar(workspace: workspace, windowController: windowController)
+				NavigatorSidebar(workspace: workspace, windowController: windowController)
+//                NavigatorSidebar(workspace: workspace, windowController: windowController)
```

Comment:
Remove this line of comment

Reply:
Yeah I definitely will do when I clean up everything. 😊